docs/tools/offline/cli-device-management.md

```diff
@@ -0,0 +1,76 @@
+### Updating Devices with Mbed CLI
+
+Mbed CLI includes features that help prepare and ship updates for devices managed through Pelion. The Mbed CLI uses the subcommands starting with `mbed device-management`, `mbed dev-mgmt` or `mbed dm` to manage devices.
+
+Start by configuring your Mbed Cloud SDK API Key, target and toolchain. Obtain the API Key from the Pelion portal.
+
+```
+$ mbed config -G CLOUD_SDK_API_KEY ak_...
+$ mbed target K64F
+$ mbed toolchain GCC_ARM
+```
+
+Next, initialize the device management feature of Mbed CLI with the following command:
+
+```
+$ mbed device-management init -d "<company domain name>" -m "<product model identifier>"
+```
+
+First, this command asks for information about your update certificate. When Mbed CLI has enough information, it creates several files:
+* A certificate in `.update-certificates/default.der`.
+* A matching private key in `.update-certificates/default.key.pem`.
+* A set of default settings in `.manifest_tool.json`.
+* Mbed Cloud settings in `.mbed_cloud_config.json`.
+* Mbed Cloud developer credentials in `mbed_cloud_dev_credentials.c`
+* Mbed Cloud update credentials in `update_defalut_resources.c`
+
+The default settings include:
+* A unique vendor identifier, based on the domain name supplied to `init`.
```

Comment:
When did I supply things to `init`?

Reply:
You supplied them to `mbed dm init` above.